editor/karma.conf.js

```diff
@@ -70,5 +77,21 @@ module.exports = function (config) {
         timeout: config.debug ? 1000000 : 10000,
       },
     },
+    parallelOptions: {
+      executors: isGithubActionsEnvironment ? cpuCores : cpuCores / 2,
+      shardStrategy: 'round-robin',
+      // shardStrategy: 'description-length'
```

Comment:
are these commented lines helpful for some future debugging?

Reply:
good question! we can keep them as "user manual" if we don't like the default sharding strategy. or I can remove it and just link to the README.md